BraveWallet/Extensions/WeiFormatter.swift

```diff
@@ -150,13 +150,13 @@ struct WeiFormatter {
       .rounded().asString(radix: outputRadix.rawValue)
   }
   
-  static func decimalToLamports(_ decimalString: String) -> UInt64? {
+  static func decimalToSplAmount(_ decimalString: String, tokenDecimals: Int) -> UInt64? {
```

Comment:
I don't think we should include `Spl` in this function name as it's used for SOL tokens too

Reply:
Resolved in [f30db1f](https://github.com/brave/brave-ios/pull/5797/commits/f30db1f20bc465c71ab738a1de129920e79622b6)